src/steps/generic.rs

```diff
@@ -171,6 +171,11 @@ pub fn run_custom_command(name: &str, command: &str, run_type: RunType) -> Resul
     run_type.execute(shell()).arg("-c").arg(command).check_run()
 }
 
+pub fn run_asdf_update(run_type: RunType) -> Result<()> {
```

Comment:
I don't think asdf supports windows, so you should wrap this with `#[cfg(unix)]`

Reply:
I followed the example as from tmux. I'm unsure if that is the correct approach, or if I should just annotate the whole method with the `#[cfg(unix)]`.

Reply:
Personally i would put the whole method under `#[cfg(unix)]` because the step will be run anyway as it's listed in the `Step` enum:
https://github.com/r-darwish/topgrade/blob/def4c3500c96c7761bf61e61631ae882184a7fb7/src/config.rs#L256
Which would actually lead to topgrade just crashing on Windows because it would try to run the step and trigger the `unreachable!` (which i think is not the correct macro to use too for this kind of error)